Asset compilation fails (Duplicate index signature)

Assets do not compile for me with fresh clone (`a63c3559b`) of `manageiq-ui-classic`. Running `bundle exec rake update:ui`, TypeScript complains about duplicate index signature:

```
ERROR in /Users/stomsa/Vývoj/ManageIQ/Repozitáře/manageiq-ui-classic/node_modules/@types/angular/jqlite.d.ts
(29,5): error TS2375: Duplicate number index signature.

ERROR in /Users/stomsa/Vývoj/ManageIQ/Repozitáře/manageiq-ui-classic/node_modules/@types/angular/jqlite.d.ts
(811,5): error TS2374: Duplicate string index signature.
```

My rather uninformed guess is that `angular-mocks` is the culprit: The error is signaled in the `/node_modules/@types/angular/jqlite.d.ts`, but there is also `/node_modules/@types/angular-mocks/node_modules/@types/angular` with the same `jqlite.d.ts` file. Maybe this is somehow causing the conflict?

With a workaround based on @himdel ’s suggestion, assets are compiled and rake command succeeds if I remove `@types/angular` and `@types/angular-mocks` from  `"devDependencies"` in `package.json`. Removing only `@types/angular-mocks` does not help.

You can see the standard output and the Yarn error log here: https://gist.github.com/Glutexo/570548c450e4474ef8b9c86819952b6d

Happens on macOS – `Darwin Kernel Version 17.4.0: Sun Dec 17 09:19:54 PST 2017; root:xnu-4570.41.2~1/RELEASE_X86_64`

--- a/package.json
+++ b/package.json
@@ -21,16 +21,10 @@
   },
   "homepage": "https://github.com/ManageIQ/manageiq#readme",
   "dependencies": {
-    "@angular/common": "~4.0.3",
-    "@angular/compiler": "~4.0.3",
-    "@angular/core": "~4.0.3",
-    "@angular/platform-browser": "~4.0.3",
-    "@angular/platform-browser-dynamic": "~4.0.3",
     "angular": "~1.6.6",
     "angular-animate": "~1.6.6",
     "angular-sanitize": "~1.6.6",
     "base64-js": "~1.2.3",
-    "core-js": "~2.4.1",
     "jquery": "~2.2.4",
     "lodash": "^4.17.4",
     "ng-redux": "^3.5.2",
@@ -42,12 +36,9 @@
     "redux-devtools-extension": "^2.13.2",
     "rxjs": "~5.3.0",
     "text-encoder-lite": "git://github.com/coolaj86/TextEncoderLite.git#e1e031b",
-    "ui-select": "0.19.8",
-    "zone.js": "~0.8.5"
+    "ui-select": "0.19.8"
   },
   "devDependencies": {
-    "@types/angular": "1.6.29",
-    "@types/angular-mocks": "^1.5.11",
     "@types/jasmine": "^2.8.6",
     "@types/jest": "^22.1.3",
     "@types/redux": "^3.6.0",

